sklearn/plot/_gridsearch_results.py

```diff
@@ -0,0 +1,143 @@
+import numpy as np
+from sklearn.plot import plot_heatmap
+
+
+def _plot_1D_results(cv_results, params, ax, xlabel, ylabel, title,
+                     fmt, xtickrotation):
+    import matplotlib.pyplot as plt
```

Comment:
@aarshayj not sure I follow. Why not put it at the top of ``plot_gridsearch_results``?